analysis/interface/topObjectSelection.h

```diff
@@ -20,6 +20,7 @@ class topObjectSelection : public nanoBase
   std::vector<TParticle> vetoElecSelection();
   std::vector<TLorentzVector> recoleps;
   std::vector<TParticle> jetSelection();
+  std::vector<TParticle> gjetSelection();
```

Comment:
can we call this genJetSelection or something similar. gjet sounds like gluon jet. Also, maybe separate it from the reco selections (i.e. move it down, not between jet and bjet selection)